packages/react-native-editor/__device-tests__/pages/editor-page.js

```diff
@@ -159,12 +160,13 @@ class EditorPage {
 
 	async getTextViewForHtmlViewContent() {
 		const accessibilityId = 'html-view-content';
-		let blockLocator = `//*[@${ this.accessibilityIdXPathAttrib }="${ accessibilityId }"]`;
+		const htmlViewLocator = isAndroid()
+			? `//*[@${ this.accessibilityIdXPathAttrib }="${ accessibilityId }"]`
+			: `//XCUIElementTypeTextView[starts-with(@${ this.accessibilityIdXPathAttrib }, "${ accessibilityId }")]`;
 
-		if ( ! isAndroid() ) {
-			blockLocator = `//XCUIElementTypeTextView[starts-with(@${ this.accessibilityIdXPathAttrib }, "${ accessibilityId }")]`;
-		}
-		return await this.driver.elementByXPath( blockLocator );
+		await waitForVisible( this.driver, htmlViewLocator );
+
+		return await this.driver.elementByXPath( htmlViewLocator );
```

Comment:
I'm thinking that we could make the `waitForVisible` function to return the element, this way we could save querying again, wdyt?

That way, we could have:
```
return await waitForVisible( this.driver, htmlViewLocator );
```

Reply:
This suggestion has been applied in https://github.com/WordPress/gutenberg/pull/39377/commits/a9298eede3ae08ca1492f0d8d1ca4e67da722ab3.